manifests/install/repos/extra.pp

```diff
@@ -30,12 +30,7 @@
   }
 
   if $ipa_authentication and $configure_ipa_repo {
-    yumrepo { 'adelton-identity':
```

Comment:
> it's an incompatible removal

Our hope is that it's not incompatible because whatever the repo was providing is in base OSes by now.